ibis/expr/scope.py

```diff
@@ -159,7 +212,7 @@ def merge_scopes(
 
 
 def make_scope(
-    op: Node, result: Any, timecontext: Optional[TimeContext] = None
+    op: Node, timecontext: Optional[TimeContext], value: Any
 ) -> 'Scope':
     """make a Scope instance, adding (op, result, timecontext) into the
```

Comment:
i find this slightly weird that `make_scope` takes a Node but Scope takes a dict. can we unify?

Reply:
Yes, now with the new constructor of Scope it makes sense to kill this make_scope entirely and use the Scope constructor only. This will simplify the usage of this class.